v1/mobileapps.yaml

```diff
@@ -270,54 +270,3 @@ paths:
                             from_backend.headers) }}'
               body: '{{from_backend.body}}'
       x-monitor: false
-
-  /mobile-text/{title}:
```

Comment:
@berndsi Yes, but those graphs only shows top-20 endpoints by request rate. To see all of them you can go to grafana-admin and change the query.